Brisanet/ion
# feat: close popconfirm when click outside and improve directive #658

## Issue Number

fix #658
fix #657

## Description

This PR change too remove firstOpen flag from button/dropdown and fix popconfirm close when clickout side.

## How to Test

You can check on button with dropdown clicking in outside element and too you can repeat this same behavior in smart table when open a popconfirm from action.

## View Storybook

## Compliance
